ex1/ex1.js

```diff
@@ -8,25 +8,27 @@
  */
 
 /////////////////////////////////////////////////////////////////////////////
-// getThirdCommaIndex accepts a string and returns the index of the third comma in that string if no comma return -1
-function getThirdCommaIndex(str,searchFor) {
+// Accepts a string and returns the index of the third comma in that string if no comma return -1
+//
+//
+
 
+function getThirdCommaIndex(str,searchFor) {
+        
+	//  i do not know how to explain the use of  -1
         var index = -1;
 
-	for (let i = 0; i < 3; i++) {
-		index = str.indexOf(searchFor, index+1);
+	for (var i = 0; i < 3; i++) {
+
+		index = str.indexOf(searchFor, index + 1);
+
+		//  If we did not find a comma in str will stop search
+	
 		if (index == -1) { 
 			break;
 		}
 	}
 	return index;
 }
 
-function testFunction(str) {
-
-	console.log(getThirdCommaIndex(str, ','));
-}
-
-testFunction('hel,lowo,rld');// === -1
-testFunction('hello,world,this,is,a,great,day');// === 16
-testFunction(',,,');// === 2
+module.exports = { getThirdCommaIndex }
```

Comment:
This is equivalent to:
`module.exports = { getThirdCommaIndex: getThirdCommaIndex }`
This exports an object with one property named getThirdCommaIndex.

Since you don't need to export anything else, you can just export the function directly:
`module.exports = getThirdCommaIndex`
The test file will have to change too, obviously.